controllers/defaults/placements.go

```diff
@@ -126,6 +126,30 @@ var (
 				getOcsToleration(),
 			},
 		},
+
+		"provider-server": {
+			Tolerations: []corev1.Toleration{
+				getOcsToleration(),
+			},
+		},
+
+		"metrics-exporter": {
+			Tolerations: []corev1.Toleration{
+				getOcsToleration(),
+			},
+		},
+
+		"csi-plugin": {
```

Comment:
we should declare keys like "csi-plugin" and "csi-provisioner" as const and reuse them to avoid introducing bugs due to misspelling.

Reply:
Done